src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -43,5 +43,10 @@ internal enum ConversionKind : byte
         IntPtr,
         InterpolatedString, // a conversion from an interpolated string to IFormattable or FormattableString
         Deconstruction, // The Deconstruction conversion is not part of the language, it is an implementation detail 
+
+        // IdentityValue is not a part of the language. 
+        // It is used by lowering to ensure that trivially reduced expressions 
+        // do not become exposed to mutations if used as receivers of struct methods.
+        IdentityValue,  
```

Comment:
If the sole intent is to force an rvalue then why not call the conversion `RValue`?  My estimation is `IdentityValue` will make code harder to read as it's not a language element and the name doesn't describe the intent.